apps/offline/signals.py

```diff
@@ -12,4 +15,5 @@ def trigger_create_thumbnail(sender, instance: Issue, created=False, **kwargs):
     Calls the create thumbnail task if an issue saved (either created or updated).
     """
     if not instance.image:
-        create_thumbnail_task.delay(issue_id=instance.id)
+        transaction.on_commit(lambda: create_thumbnail_task.delay(issue_id=instance.id))
```

Comment:
Why is this needed?

Reply:
I experienced a race condition where the celery task executed before the dB transaction committed when running on AWS. So when the task tried to get the object it couldn't because it wasn't in the dB yet. This resolves that bh waiting for the transaction to be committed.